homeassistant/components/zwave_js/icons.json

```diff
@@ -18,7 +23,51 @@
           "dead": "mdi:robot-dead",
           "unknown": "mdi:help-rhombus"
         }
+      },
+      "can": {
+        "default": "mdi:car-brake-alert"
+      },
+      "nak": {
+        "default": "mdi:hand-back-left-off"
+      },
+      "timeout_ack": {
+        "default": "mdi:ear-hearing-off"
+      },
+      "timeout_callback": {
+        "default": "mdi:timer-sand-empty"
+      },
+      "timeout_response": {
+        "default": "mdi:timer-sand-empty"
+      },
+      "successful_messages": {
+        "default": "mdi:check"
+      },
+      "messages_dropped": {
+        "default": "mdi:trash-can"
+      },
+      "successful_commands": {
+        "default": "mdi:check"
+      },
+      "commands_dropped": {
```

Comment:
Please sort them

Reply:
went ahead and did so but is this a standard? If so, shouldn't we have something in pre-commit that enforces that?